prow/config.yaml

```diff
@@ -10281,6 +10294,19 @@ periodics:
       - --bare
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180511-51cf02e75-master
 
+- interval: 6h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-gce-kubedns-performance
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
```

Comment:
these both need to be quoted, we fixed some things and technically to be valid yaml they can't be unquoted (otherwise they are a bool instead of a string), this is causing the presubmit failures, ditto for the other job labels